prog/dftb+/lib_dftb/dispdftd4.F90

```diff
@@ -660,7 +660,7 @@ subroutine dispersionGradient(calc, iAtFirst, iAtLast, nNeighbour, neigh, &
     real(dp) :: f6, df6, f8, df8, f10, df10
 
     !$omp parallel do default(none) schedule(runtime) &
-    !$omp reduction(+:energies, gradients, sigma, dEdq, dEdcn) &
+    !$omp shared(energies, gradients, sigma, dEdq, dEdcn) &
```

Comment:
I am not sure. Wouldn't we get possible race conditions, when `energies(iAt2f)` and `gradients(:, iAt2f)` is accessed below? I fear, that this would only work with a full (symmetric) neighbour list.